### ISSUE TITLE
FGVC code

### ISSUE THREAD

**Issue by khawar-islam** [3neutronstar/GuidedMixup#4]
Dear @3neutronstar 

Would it be possible for you to give me a favor? As I have searched a lot on GitHub regarding FGVC code and mostly codes are utilizing pre-trained model to fine-tune on FGVC dataset. When i trained from scratch the accuracy significantly dropped, code you please your FGVC codebase?

Regards,
Khawar

**Comment by khawar-islam** [comment]
Dear @3neutronstar When I trained resnet50 (Vanilla) model from scratch on Stanford Car dataset, the model achieved 80.92% results and in your paper  resnet50 (Vanilla) accuracy is 85.52% which is quite different. That is my problem I am facing right now

**Comment by 3neutronstar** [comment]
Dear @khawar-islam, 

Thank you for your suggestion. I will upload FGVC implementation soon.

**Comment by khawar-islam** [comment]
Dear @3neutronstar 

thank you very much, I will be waiting for your code
